docs/calib.rst

```diff
@@ -0,0 +1,141 @@
+#. Invert the spectral dispersion to find the parameter (:math:`t`).
+#. Evaluate the spectral trace with the parameter (:math:`t`).
+
+.. note::
+	For linear dispersion models, this inversion can be done analytically.  For higher-order polynomials, ``slitlessutils`` inverts using `Halley's Method <https://en.wikipedia.org/wiki/Halley%27s_method>`_.
+
+
+.. _flatfield:
+
+Flat Field
+----------
+
+The flat-field corrects for differences in the pixel-to-pixel sensitivity, and is derived by observing a suitably flat illumination pattern.  Importantly, this correction is wavelength-dependent, but the wavelength covered by a WFSS image pixel will depend on the *undispersed position* :math:`(x_0,y_0)`.  Therefore, the WFSS images are not flat-fielded by the calibration pipelines, and so it must be accounted for in the extraction/simulation processes.  ``Slitlessutils`` implements the wavelength-dependent flat field as a polynomial in wavelength:
+
+.. math::
+	F_{x,y}(\lambda) = \sum_{k=0} F_{x,y}\,w(\lambda)^k
+
+where 
+
+.. math::
+	w(\lambda) = \left\{\begin{array}{ll}
+	0 & \text{for } \lambda<\lambda_0 \\
+	\frac{\lambda-\lambda_0}{\lambda_1-\lambda_0} & \text{for } \lambda_0\leq\lambda\leq\lambda_1 \\
+	0 & \text{for } \lambda_1<\lambda\end{array}\right.
+
+and the parameters :math:`\lambda_0, \lambda_1` are the lower and upper bounds (respectively) for which the flat-field cube is defined.  See :numref:`flatfield` below for a schematic layout of this polynomial flat field.  Additionally, users may also specify a *gray flat* (typically derived from a direct image flat field, which is effectively just a single level in :numref:`flatfieldexample`) or a unity flat (effectively ignoring the flat-field correction entirely).  See:
+
+* Unity flat field: :class:`~slitlessutils.core.wfss.config.UnityFlatField()`
+* Gray flat field: :class:`~slitlessutils.core.wfss.config.ImageFlatField()`
+* Polynomial flat field: :class:`~slitlessutils.core.wfss.config.PolynomialFlatField()`
+* factory function to load these: :func:`~slitlessutils.core.wfss.config.load_flatfield()`
+
+
+
+.. _flatfieldexample:
+.. figure:: images/FITSflat.png
+   :width: 600
+   :alt: The schematic layout of a fits flat-field cube.
+
+   A schematic layout of the fits flat-field cube.  This figure is taken from the `aXe manual <https://hstaxe.readthedocs.io/en/latest/>`_ and reproduced here courtesy of Nor Pirzkal.
+
+
+
+.. _sensitivity:
+
+Sensitivity Curves
+------------------
+
+The :term:`sensitivity curve` provides the conversion between detector units (usually :math:`e^-/s`) to physical units (usually :math:`erg/s/cm^2/Å`), which depends on spectral order by **NOT** on spatial extent as that is addressed by the :ref:`flat-field <flatfield>`. This can be thought of as a wavelength-dependent :term:`zeropoint` in flux units.  :numref:`senscurves` shows the sensitivity curves for several :term:`grism` and :term:`prism` modes for several HST instruments.
```

Comment:
```suggestion
The :term:`sensitivity curve` provides the conversion between detector units (usually :math:`e^-/s`) to physical units (usually :math:`erg/s/cm^2/Å`), which depends on spectral order but**NOT** on spatial extent, as that is addressed by the :ref:`flat-field <flatfield>`. This can be thought of as a wavelength-dependent :term:`zeropoint` in flux units.  :numref:`senscurves` shows the sensitivity curves for several :term:`grism` and :term:`prism` modes for several HST instruments.
```